go.mod

```diff
@@ -299,3 +299,5 @@ require (
 	k8s.io/klog/v2 v2.120.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/mitchellh/mapstructure => github.com/go-viper/mapstructure v1.6.0
```

Comment:
T
I think this breaks `go run` which we use in CI for the rules and profiles repo. Why not replace the imports?

Reply:
From https://github.com/go-viper/mapstructure/releases/tag/v1.6.0:

> This is a maintenance release after forking the library to make some of the fixes available without major changes.
>
> Usage:
>
> ```
> go get github.com/mitchellh/mapstructure
> go mod edit -replace github.com/mitchellh/mapstructure=github.com/go-viper/mapstructure
> go mod tidy
> ```